src/peft/tuners/boft/layer.py

```diff
@@ -77,9 +78,6 @@ def get_fbd_cuda():
     if _FBD_CUDA is not None:
         return _FBD_CUDA
 
-    # This import initializes cuda context and should thus be local, see issue 1877
-    from torch.utils.cpp_extension import load
```

Comment:
Could you please undo this change? The import should be local. Maybe merging with/rebasing on the latest main is sufficient.

Reply:
done